kolibri/utils/server.py

```diff
@@ -322,3 +323,39 @@ def get_urls(listen_port=None):
         return STATUS_RUNNING, urls
     except NotRunning as e:
         return e.status_code, []
+
+
+def installation_type():
```

Comment:
Maybe moving this to `kolibri.utils.version` since it's not supposed to depend on anything external, except maybe a `VERSION_PKG` well-known file location? (see below suggestion)

Reply:
@jredrejo I still think that all this logic pertains a kind of versioning of the distributed package. I think it should be moved to `kolibri.utils.version`